includes/Modules/Idea_Hub.php

```diff
@@ -162,15 +162,14 @@ public function register_persistent() {
 		add_filter(
 			'display_post_states',
 			function( $post_states, $post ) {
-				if ( 'draft' !== $post->post_status ) {
-					return $post_states;
-				}
 				$idea = $this->get_post_idea( $post->ID );
 				if ( is_null( $idea ) ) {
 					return $post_states;
 				}
-				/* translators: %s: Idea Hub Idea Title */
-				$post_states['draft'] = sprintf( __( 'Idea Hub Draft “%s”', 'google-site-kit' ), $idea['text'] );
+				$post_states[ $post->post_status ] = '' === $post->post_title ?
```

Comment:
I think we may need to reconsider the implementation here as it would be currently overwriting the default states for every post status for all Idea Hub posts for which there are quite a few https://github.com/WordPress/WordPress/blob/baa81b67d19e29be29d0ac49c9bede316ab8f80a/wp-admin/includes/template.php#L2183-L2211

Ideally this would be an additive annotation rather than taking over the state entirely for IH posts. Thoughts @felixarntz?

Reply:
@aaemnnosttv @ivankruchkoff Good point, I agree. Let's do it like this:
* Only for the `draft` status specifically, if the title is empty, use the existing "Idea Hub Draft xyz".
* Otherwise, if there is already content for the status, combine the original content with "inspired by Idea Hub" e.g. like `sprintf( __( '%s, inspired by Idea Hub', 'google-site-kit' ), $original_content )`.
* Otherwise (i.e. not a `draft` and there is no original content for the status, set it to "inspired by Idea Hub".

Reply:
Updated in https://github.com/google/site-kit-wp/pull/4061/commits/2204098e7c21907855e5c98c95e0b6ee4abf8ebc